website/content/specification/syntax/constraints.md

```diff
@@ -164,6 +164,21 @@ And the following document.
 The expect constraint would pass for each `sibling` in the `parent` named "p1", and would fail for each `sibling` in the `parent` named "p2".
 
 
+## `is-unique` constraints
```

Comment:
I know we are discussing this in Element so I will help, but I think we need to fix how you committed the changes that are based on `is-unique` and how you adapted from from that branch into the `matches` work that should be here. Did you fork off of the `Arminta-Jenkins-NIST:document-is-unique-constraint` branch for 512 to start? It seems likely. I can help you with that in a bit.